testdata/project-v3-addon/config/manager/manager.yaml

```diff
@@ -26,6 +26,8 @@ spec:
     spec:
       securityContext:
         runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
```

Comment:
/hold

We need to check if it will works well from 1.19

Reply:
Yeah, it works for 1.19+. The `SeccompDefault` featureGate provided in 1.22+ is a global gate, which enables all Pods to use `seccompProfile= RuntimeDefault`, so we don't have to care about it here.

Reply:
For the default scaffold, we want to ensure that all scaffold by the tool is restrictive (we do not want really on the cluster-admin to follow the good practice)

However, is nice to know that the cluster-admin is also able to enable that for all Pods. It means no harm to ensure as well in case the admin does not follow it. 

Thank you a lot.